### ISSUE TITLE
Regression 1.17.1 -> 1.18.0: Uncaught TypeError Argument #1 ($text) must be of type string, null given

### ISSUE THREAD

**Issue by dregad** [PHPOffice/PhpSpreadsheet#2294]
This is:
```
- [x] a bug report
- [ ] a feature request
- [x] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?

Xslx spreadsheet saved without errors. 

### What is the current behavior?

Program crashes when saving spreadsheet with combination of
- AutoSize method AUTOSIZE_METHOD_EXACT
- Auto sized column
- Empty cell with style applied in that column

```
Fatal error: Uncaught TypeError: Argument 1 passed to PhpOffice\PhpSpreadsheet\Shared\Font::getTextWidthPixelsExact() must be of the type string, null given, called in /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Shared/Font.php on line 254 and defined in /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Shared/Font.php:277
Stack trace:
#0 /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Shared/Font.php(254): PhpOffice\PhpSpreadsheet\Shared\Font::getTextWidthPixelsExact()
#1 /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/Worksheet.php(766): PhpOffice\PhpSpreadsheet\Shared\Font::calculateColumnWidth()
#2 /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Writer/Xlsx/Worksheet.php(369): PhpOffice\PhpSpreadsheet\Worksheet\Worksheet->calculateColumnWidths()
#3 /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadshee in /tmp/test/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Shared/Font.php on line 277
```

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php
use PhpOffice\PhpSpreadsheet\IOFactory;
use PhpOffice\PhpSpreadsheet\Spreadsheet;
use PhpOffice\PhpSpreadsheet\Shared\Font;
use PhpOffice\PhpSpreadsheet\Style\Alignment;

require_once 'vendor/autoload.php';

Font::setTrueTypeFontPath('/usr/share/fonts/truetype/msttcorefonts/');
Font::setAutoSizeMethod(Font::AUTOSIZE_METHOD_EXACT);

$xl = new Spreadsheet();
$xl->getDefaultStyle()->getFont()->setName('Arial')->setSize(12);

$xlWS = $xl->getSheet(0);
$xlWS->fromArray(['Test 1', 123], null, 'A1');
$xlWS->fromArray(['Test 2', null], null, 'A2');
$xlWS->getStyle('B2')->getAlignment()->setHorizontal(Alignment::HORIZONTAL_LEFT);

foreach ($xlWS->getColumnIterator() as $col) {
    $xlWS->getColumnDimension($col->getColumnIndex())->setAutoSize(true);
}

$xlWrite = IOFactory::createWriter($xl, 'Xlsx');
$xlWrite->setPreCalculateFormulas(false);
$xlWrite->save('/tmp/test.xslx');
```

### Which versions of PhpSpreadsheet and PHP are affected?

This worked just fine until 1.17.1.
Problem started occurring with 1.18.0

Tested with PHP 7.4 and PHP 8.0.

**Comment by dregad** [comment]
I just realized that this issue has already been fixed in master branch (see #2167). Sorry for not checking before.

Is there any chance you guys could release 1.18.1 including this fix ? Thanks in advance !

**Comment by oleibman** [comment]
Fixed in [1.19.0](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/1.19.0)
